### PR TITLE
Update setup_default_docker task

### PR DESCRIPTION
**Pull request by elyezer**
Match the steps recommended on the Docker prerequisites section from the
docs.

Drop pulling busybox image and running the docker ps command if any
issue happens the manage_daemon will fail.

### CODE REVIEW

**Comment by omaciel** [comment]
**NOTE**: this will now automatically configure Docker to be used either as internal (unix socket) and external (http) compute resource.

ACK

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -218,13 +217,6 @@ def setup_default_docker():
     # https://github.com/docker/docker/issues/2758
     manage_daemon('restart', 'docker', pty=(os_version >= 7))
 
-    # Check that things look good
-    run('docker ps')
```
Comment (sthirugn) [inline automation_tools/__init__.py]: We can leave this just to make sure that docker is really installed? (may be with warn_only=True)

**Comment by sthirugn** [comment]
ACK pending comment

File: automation_tools/__init__.py (hunk 1 above)
Comment (elyezer) [inline automation_tools/__init__.py]: @sthirugn as the commit message states, manage_daemon will fail if docker is not installed.

Comment (sthirugn) [inline automation_tools/__init__.py]: ok :+1:

**Pull request merged**